runtime/moonbase/src/precompiles.rs

```diff
@@ -92,6 +99,19 @@ parameter_types! {
 
 type EthereumPrecompilesChecks = (AcceptDelegateCall, CallableByContract, CallableByPrecompile);
 
+// Pallet-xcm precompile types.
+// Type that converts AssetId into Location
+type AssetIdToLocationManager = AsAssetType<AssetId, AssetType, AssetManager>;
+
+// The pallet-balances address is identified by 2050
+type SingleAddressMatch = SingleAddressMatcher<AccountId, 2050, Balances>;
```

Comment:
I think we should replace this by a constant to make the maintainability easier and less error prone.

```rs

const ERC20_BALANCES_PRECOMPILE: u64 = 2050;
type SingleAddressMatch = SingleAddressMatcher<AccountId, ERC20_BALANCES_PRECOMPILE, Balances>;
...
PrecompileAt<
  AddressU64<ERC20_BALANCES_PRECOMPILE>,
  Erc20BalancesPrecompile<R, NativeErc20Metadata>,
  (CallableByContract, CallableByPrecompile),
>,
```